### PR TITLE
[16.0][IMP]helpdesk_mgmt: Add commercial partner on ticket

### PR DESCRIPTION
**Pull request by flotho**

### CODE REVIEW

**Comment by flotho** [comment]
ping @vincent-hatakeyama , any review possible on this one ?

Regards

File: helpdesk_mgmt/models/helpdesk_ticket.py [hunk 1]
```diff
@@ -59,6 +59,12 @@ def _read_group_stage_ids(self, stages, domain, order):
         domain="['|',('team_ids', '=', team_id),('team_ids','=',False)]",
     )
     partner_id = fields.Many2one(comodel_name="res.partner", string="Contact")
+    commercial_partner_id = fields.Many2one(
+        "res.partner",
```
Comment (dalonsod) [inline helpdesk_mgmt/models/helpdesk_ticket.py]: Not sure, but for `Many2one` related fields no `comodel_name` needs to be added, could you check?

**Review by dalonsod — CHANGES_REQUESTED** [review]
See comment.

Please add to commit and PR titles addon name after `[IMP]` label, so this change could be better identified.

**Comment by marcelsavegnago** [comment]
> See comment.
> 
> Please add to commit and PR titles addon name after `[IMP]` label, so this change could be better identified.

ex.: [IMP] helpdesk_mgmt: bla bla bla bla ...

File: helpdesk_mgmt/models/helpdesk_ticket.py (hunk 1 above)
Comment (DorianMAG) [reply helpdesk_mgmt/models/helpdesk_ticket.py]: > Not sure, but for `Many2one` related fields no `comodel_name` needs to be added, could you check?

Seems not necessary comodel_name
https://www.odoo.com/documentation/16.0/developer/reference/backend/orm.html#odoo.fields.Many2one

"comodel_name – name of the target model Mandatory except for related or extended fields."

**Review by JulienMartinez — APPROVED** [review]
LGTM

File: helpdesk_mgmt/models/helpdesk_ticket.py (hunk 1 above)
Comment (dalonsod) [reply helpdesk_mgmt/models/helpdesk_ticket.py]: Well, then it should be removed IMO

**Review by GuiPrime — APPROVED** [review]
LGTM

File: helpdesk_mgmt/models/helpdesk_ticket.py (hunk 1 above)
Comment (flotho) [reply helpdesk_mgmt/models/helpdesk_ticket.py]: done

**Comment by flotho** [comment]
> > See comment.
> > Please add to commit and PR titles addon name after `[IMP]` label, so this change could be better identified.
> 
> Sample: [IMP] helpdesk_mgmt: bla bla bla bla ...

done

**Review by dalonsod — APPROVED** [review]
:+1: and don't forget to change the PR title as well

**Review by DorianMAG — APPROVED** [review]
LGTM

**Comment by flotho** [comment]
> 👍 and don't forget to change the PR title as well

re done

**Review by marcelsavegnago — APPROVED** [review]
LGTM

**Review by vincent-hatakeyama — APPROVED** [review]

**Pull request merged**